examples/python/clients/httpx/.env-example

```
@@ -0,0 +1,3 @@
+PRIVATE_KEY=your_evm_private_key_here
```

Suggest renaming to EVM_PRIVATE_KEY in anticipation of svm to be added soon
```suggestion
EVM_PRIVATE_KEY=your_evm_private_key_here
```